packages/lib/src/components/UPI/components/UPIComponent/UPIComponent.tsx

```diff
@@ -69,23 +77,25 @@ export default function UPIComponent({ defaultMode, onChange, onUpdateMode, payB
 
             setSelectedApp(app);
             setIsValid(true);
+            setStatus('success');
```

Comment:
Do we need to setStatus to `success` here? Does it do something?

`success` in general means that the payment was completed, but in this particular case it means that one app from the intent list was selected. It seems a bit weird to me